src/js/veteran-id-card/containers/Main.jsx

```diff
@@ -11,13 +11,29 @@ class Main extends React.Component {
     super(props);
     this.handleSubmit = this.handleSubmit.bind(this);
     this.renderButton = this.renderButton.bind(this);
+    this.submitVicForm = this.submitVicForm.bind(this);
   }
 
   handleSubmit(e) {
     e.preventDefault();
     this.props.initiateIdRequest();
   }
 
+  submitVicForm() {
+    const vicForm = document.createElement('form');
```

Comment:
I think it would be more React-ish to render this in the render method below, and trigger the submit in this method.

So you could put this in the jsx return in render():

```jsx
{!!this.props.vicUrl && 
  <form id="vicForm" method="POST" action={this.props.vicUrl}>
    {Object.entries(this.props.traits).map(([key, value] => (
      <input type="hidden" name={key} key={key} value={value}/>
    )}
  </form>
}
```

And have the submit in a lifecycle method:

```js
componentDidUpdate() {
  if (this.props.vicUrl) {
    document.getElementById('vicForm').submit();
  }
}
```